docs/deployments/README.md

```diff
@@ -0,0 +1,25 @@
+# Deployment examples
+The eUPF can be integrated with different 5G Core implementations in different scenarios.
+
+eUPF pod outbound connection is pure routed at the node. There is no address translation inside pod, so we avoid such lack of throughtput.
+
+>If you need Network Address Translation (NAT) function for subscriber's egress traffic, see the chapter [about NAT](#option-nat-at-the-node) below.
```

Comment:
In case of Calico BGP(BIRD) NAT is provided as well. NAT clarification nore should be more clear.